code/modules/mining/abandoned_crates.dm

```diff
@@ -0,0 +1,213 @@
+		if(83 to 84)
+			new /obj/item/toy/katana(src)
+		if(85 to 86)
+			new /obj/item/weapon/defibrillator/compact(src)
+		if(87) //1% chance
+			new /obj/item/weed_extract(src)
+		if(88)
+			new /obj/item/organ/brain(src)
+		if(89)
+			new /obj/item/organ/brain/alien(src)
+		if(90)
+			new /obj/item/organ/heart(src)
+		if(91)
+			new /obj/item/device/soulstone/anybody(src)
+		if(92)
+			new /obj/item/weapon/katana(src)
+		if(93)
+			new /obj/item/weapon/dnainjector/xraymut(src)
+		if(94)
+			new /obj/item/weapon/storage/backpack/clown(src)
+			new /obj/item/clothing/under/rank/clown(src)
+			new /obj/item/clothing/shoes/clown_shoes(src)
+			new /obj/item/device/pda/clown(src)
+			new /obj/item/clothing/mask/gas/clown_hat(src)
+			new /obj/item/weapon/bikehorn(src)
+			new /obj/item/toy/crayon/rainbow(src)
+			new /obj/item/weapon/reagent_containers/spray/waterflower(src)
+		if(95)
+			new /obj/item/clothing/under/rank/mime(src)
+			new /obj/item/clothing/shoes/sneakers/black(src)
+			new /obj/item/device/pda/mime(src)
+			new /obj/item/clothing/gloves/color/white(src)
+			new /obj/item/clothing/mask/gas/mime(src)
+			new /obj/item/clothing/head/beret(src)
+			new /obj/item/clothing/suit/suspenders(src)
+			new /obj/item/toy/crayon/mime(src)
+			new /obj/item/weapon/reagent_containers/food/drinks/bottle/bottleofnothing(src)
+		if(96)
+			new /obj/item/weapon/hand_tele(src)
+		if(97)
+			new /obj/item/seeds/gatfruit(src)
+		if(98)
+			new /obj/item/weapon/katana/cursed(src)
+		if(99)
+			new /obj/item/weapon/storage/belt/champion(src)
+			new /obj/item/clothing/mask/luchador(src)
+		if(100)
+			new /obj/item/clothing/head/bearpelt(src)
+
+/obj/structure/closet/crate/secure/loot/togglelock(mob/user as mob)
+	if(locked)
+		user << "<span class='notice'>The crate is locked with a Deca-code lock.</span>"
+		var/input = input(usr, "Enter [codelen] digits.", "Deca-Code Lock", "") as text
+		if(in_range(src, user))
+			if (input == code)
+				user << "<span class='notice'>The crate unlocks!</span>"
+				locked = 0
+				overlays.Cut()
+				overlays += greenlight
+			else if (input == null || length(input) != codelen)
+				user << "<span class='notice'>You leave the crate alone.</span>"
+			else
+				user << "<span class='warning'>A red light flashes.</span>"
+				lastattempt = input
+				attempts--
+				if (attempts == 0)
+					user << "<span class='danger'>The crate's anti-tamper system activates!</span>"
+					var/turf/T = get_turf(src.loc)
+					explosion(T, 0, 0, 0, 1)
+					qdel(src)
+					return
+		else
+			user << "<span class='notice'>You attempt to interact with the device using a hand gesture, but it appears this crate is from before the DECANECT came out.</span>"
+			return
+	else
+		return ..()
+
+/obj/structure/closet/crate/secure/loot/attackby(obj/item/weapon/W as obj, mob/user as mob)
+	if(locked)
+		if (istype(W, /obj/item/weapon/card/emag))
+			user << "<span class='danger'>The crate's anti-tamper system activates!</span>"
```

Comment:
what

i spent valuable TCs on this crap why should I get penalized for using it